MANIFEST.in

```diff
@@ -40,6 +37,8 @@ global-exclude .DS_Store
 global-exclude .git*
 global-exclude \#*
 
+prune pandas/tests/io/parser/data
```

Comment:
maybe add a comment here why this is excluded. could maybe open an issue about csv_dir_path fixture and reference it here.